packages/x-codemod/src/v6.0.0/data-grid/remove-disableExtendRowFullWidth-prop/actual.spec.js

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { DataGrid } from '@mui/x-data-grid';
```

Comment:
If we are going with correctness with all the imports. 🙈 
```suggestion
import { DataGrid } from '@mui/x-data-grid';
import { DataGridPro } from '@mui/x-data-grid-pro';
import { DataGridPremium } from '@mui/x-data-grid-premium';
```

But also no one would be mad if this file only had the following 🙈 
```tsx
<>
  <DataGrid disableExtendRowFullWidth />
  <DataGridPro disableExtendRowFullWidth />
  <DataGridPremium disableExtendRowFullWidth />
</>
```